### PR TITLE
FIX: ci invalid cache

### PR DESCRIPTION
**Pull request by martins0n**
## Before submitting (must do checklist)

- [ ] Did you read the [contribution guide](https://github.com/tinkoff-ai/etna/blob/master/CONTRIBUTING.md)?
- [ ] Did you update the docs? We use Numpy format for all the methods and classes.
- [ ] Did you write any new necessary tests?
- [x] Did you update the [CHANGELOG](https://github.com/tinkoff-ai/etna/blob/master/CHANGELOG.md)?

### CODE REVIEW

File: .github/workflows/docs-on-pr.yml [hunk 1]
```diff
@@ -23,16 +23,14 @@ jobs:
...
            path: .venv
-           key: venv-${{ runner.os }}-${{ hashFiles('**/poetry.lock') }}
-
+           key: venv-${{ runner.os }}-docs-${{ hashFiles('**/poetry.lock') }}
        - name: Install dependencies
          if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
          run: |
-           poetry install -E all -vv
-       - name: Install docs dependencies
+           poetry install -E "all docs" -vv
```
Comment (alex-hse-repository) [inline .github/workflows/docs-on-pr.yml:30]: Why do we need to install "all" dependencies just to build the docs?

File: .github/workflows/notebooks.yml [hunk 2]
```diff
@@ -28,16 +28,7 @@ jobs:
...
-        id: cached-poetry-dependencies
-        uses: actions/cache@v2
-        with:
-          path: .venv
-          key: venv-${{ runner.os }}-${{ hashFiles('**/poetry.lock') }}
-
       - name: Install dependencies
-        if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
         run: |
           poetry install -E all -vv
```
Comment (alex-hse-repository) [inline .github/workflows/notebooks.yml:33]: May be the same move with installing jupyter and release here?

File: .github/workflows/notebooks.yml [hunk 3]
```diff
@@ -28,16 +28,7 @@ jobs:
         with:
           virtualenvs-create: true
           virtualenvs-in-project: true
-
-      - name: Load cached venv
```
Comment (alex-hse-repository) [inline .github/workflows/notebooks.yml:32]: Why did you decide not to use cache here?

**Review by alex-hse-repository — CHANGES_REQUESTED** [review]

File: .github/workflows/docs-on-pr.yml (hunk 1 above)
Comment (martins0n) [reply .github/workflows/docs-on-pr.yml:30]: New behavior - poetry remove all dependencies from previous steps. So you need to specify all extras at once

File: .github/workflows/notebooks.yml (hunk 2 above)
Comment (martins0n) [reply .github/workflows/notebooks.yml]: Fixed

**Review by alex-hse-repository — APPROVED** [review]

**Pull request merged**